samples/sample_encode/src/pipeline_encode.cpp

```diff
@@ -1290,7 +1290,7 @@ mfxStatus CEncodingPipeline::InitFileWriters(sInputParams *pParams)
     return sts;
 }
 
-#if (defined(_WIN64) || defined(_WIN32)) && (MFX_VERSION >= MFX_VERSION_NEXT)
+#if (defined(_WIN64) || defined(_WIN32)) && (MFX_VERSION >= 1031)
```

Comment:
why version check is here?! Just few lines below in a function you have #if (MFX_VERSION >= 1027) - it does not make sense!

Reply:
To split 12-bit enabling from discrete adapters.